### DISCUSSION TITLE
Converting the St.Omer demo to ODF

### DISCUSSION THREAD

**Discussion by oleg68** [GrandOrgue/OdfEdit#9] [Q&A] [answered]
When I trying to convert [The St. Omer demo sampleset from Sonus Paradisi](https://www.sonusparadisi.cz/en/organs/france/st-om/st-omer-surround-sample-set.html)
ODFEdit writes "Building the GrandOrgue control objects..." and does nothing more.

Both linux and windows versions.

**Comment by eturpault** [comment]
Thanks for the feedback, I add to my To Do list to look at this sample set.
Have you been able to convert other sample sets without issue ?

**Reply by oleg68** [reply]
This is the second sample set I'm trying to convert.

Earlier I tried to convert Buckeburg with the issue with case in filenames. You promised to fix it but no new version appear in the repository.

**Reply by eturpault** [reply]
I am trying to add noises stops before to deliver a new version, but it is taking me more time than expected, it is quite tricky...

**Comment by eturpault** [comment]
@oleg68 : I see that there is no demo version available for this sample set. Could you share me the ODF xml file that you tested ? Thanks.

**Reply by oleg68** [reply]
>  there is no demo version available 

It is there. Look at the `Demo version download` pattern on the page I referrenced in the first post.

**Reply by eturpault** [reply]
Ok understood, thanks.

**Comment by eturpault** [comment, marked as answer]
@oleg68 : Here is the ODF for the St Omer sample set that I generated with OdfEdit v2.1 on which I am working (not yet published, ODF to place in the folder OrganDefinitions of the sample set). It is an interresting sample set for test, thank you. You will notice that the "GO 16" coupler is acting on all the other manuals in addition to the GO manual, I will try to improve this in the future, it is due to the fact that it should have conditional effect on the manuals depending on the activation state of the couplers toward these manuals.
[St. Omer - Surround DEMO.organ.zip](https://github.com/GrandOrgue/ODFEdit/files/10429155/St.Omer.-.Surround.DEMO.organ.zip)

**Reply by oleg68** [reply]
Thank you very much for the ODF. It works fine.
